pysatMissions/instruments/missions_skyfield.py

```diff
@@ -0,0 +1,314 @@
+        Object containing satellite data
+    meta : pysat.Meta
+        Object containing metadata such as column names and units
+
+    Note
+    ----
+    * Routine is directly called by pysat and not the user.
+    * Altitude accuracy expected to be on the order of 10 km in Low Earth Orbit.
+      Efforts to improve accuracy documented under issue #79.
+
+
+    Example
+    -------
+    ::
+
+          tle1='1 25544U 98067A   18135.61844383  .00002728  00000-0  48567-4 0  9998'
+          tle2='2 25544  51.6402 181.0633 0004018  88.8954  22.2246 15.54059185113452'
+          inst = pysat.Instrument('pysat', 'sgp4', tle1=tle1, tle2=tle2)
+          inst.load(2018, 1)
+
+    """
+
+    # TLEs (Two Line Elements for ISS)
+    # format of TLEs is fixed and available from wikipedia...
+    # lines encode list of orbital elements of an Earth-orbiting object
+    # for a given point in time
+    line1 = '1 25544U 98067A   18135.61844383  .00002728  00000-0  48567-4 0  9998'
+    line2 = '2 25544  51.6402 181.0633 0004018  88.8954  22.2246 15.54059185113452'
+
+    # If provided, use user-specified TLEs.  Otherwise use ISS defaults above.
+    if tle1 is not None:
+        line1 = tle1
+    if tle2 is not None:
+        line2 = tle2
+
+    if (num_samples is None) or one_orbit:
+        # Calculate one day of samples for default
+        num_samples = len(pds.date_range('2018/1/1', '2018/1/2',
+                                         freq=cadence)) - 1
+
+    # Extract list of times from filenames and inst_id
+    times, index, dates = ps_meth.generate_times(fnames, num_samples,
+                                                 freq=cadence)
+    # Calculate epoch for orbital propagator
+    epoch_days = (epoch - dt.datetime(1949, 12, 31)).days
+    jd, _ = sapi.jday(dates[0].year, dates[0].month, dates[0].day, 0, 0, 0)
+
+    if inclination is not None:
+        # If an inclination is provided, specify by Keplerian elements
+        eccentricity, mean_motion = orbits.convert_to_keplerian(alt_periapsis,
+                                                                alt_apoapsis)
+        satrec = sapi.Satrec()
+        # According to module webpage, wgs72 is common
+        satrec.sgp4init(sapi.WGS84, 'i', 0, epoch_days, bstar, 0, 0,
+                        eccentricity, np.radians(arg_periapsis),
+                        np.radians(inclination), np.radians(mean_anomaly),
+                        mean_motion, np.radians(raan))
+    else:
+        # Otherwise, use TLEs
+        satrec = sapi.Satrec.twoline2rv(line1, line2, sapi.WGS84)
+        mean_motion = satrec.mm
+
+    if one_orbit:
+        ind = times <= (2 * np.pi / mean_motion * 60)
+        times = times[ind]
+        index = index[ind]
+
+    skytimes = skyapi.load.timescale().from_datetimes(index.tz_localize('UTC'))
+
+    esat = skyapi.EarthSatellite.from_satrec(satrec, skyapi.load.timescale())
+    # Generate object with info for all times
+    sat_obj = esat.at(skytimes)
+
```

Comment:
This is where we should put that this is the TEME ECI coords?
Skyfield docs say True Equator and Equinox of date, is that the same as true equator and mean equinox?